ArduPlane/mode_autotune.cpp

```diff
@@ -0,0 +1,24 @@
+#include "mode.h"
+#include "Plane.h"
+
+bool ModeAutoTune::_enter()
+{
+    plane.throttle_allows_nudging = false;
+    plane.auto_throttle_mode = false;
+    plane.auto_navigation_mode = false;
+    plane.autotune_start();
+
+    return true;
+}
+
+void ModeAutoTune::_exit()
+{
+    // restore last gains
+    plane.autotune_restore();
```

Comment:
Functional change, this should be `quadplane.qautotune.stop();`

Reply:
I don't understand.